src/main/java/org/apache/commons/lang3/text/translate/CharSequenceTranslator.java

```diff
@@ -119,7 +127,8 @@ public final void translate(final CharSequence input, final Writer out) throws I
      * @return CharSequenceTranslator merging this translator with the others
      */
     public final CharSequenceTranslator with(final CharSequenceTranslator... translators) {
-        final CharSequenceTranslator[] newArray = new CharSequenceTranslator[translators.length + 1];
+        @SuppressWarnings("value:assignment.type.incompatible") // translators.length + 1 has minimum value 1
+        final CharSequenceTranslator @MinLen(1) [] newArray = new CharSequenceTranslator[translators.length + 1];
```

Comment:
This looks like a bug in the Value Checker. Can you write a short test case and submit a bug?